erpnext/projects/doctype/project/project.js

```diff
@@ -130,6 +150,9 @@ function open_form(frm, doctype, child_doctype, parentfield) {
 		new_child_doc.parent = new_doc.name;
 		new_child_doc.parentfield = parentfield;
 		new_child_doc.parenttype = doctype;
+		if (doctype === "Timesheet") {
```

Comment:
Which other doctype is calling this one?

Reply:
Purchase Order and other sales related doctype